src/cmd/run.js

```diff
@@ -104,6 +109,15 @@ export default async function run(
     noReload = true;
   }
 
+  if (
+    watchFile &&
+    fs.existsSync(watchFile) &&
+    fs.lstatSync(watchFile).isDirectory()
+  ) {
+    throw new UsageError(`The directory "${watchFile}" cannot be ` +
```

Comment:
This check is not strictly necessary, but it will probably help against common mistakes.

Let's move this whole check to the `src/watcher.js`, right before `watchedFiles.push(watchFile);`.
And use `isFile()` instead of `isDirectory`. That would also defend against cases where the file path refers to non-files that are not directories, such as symlinks.